src/natcap/invest/validation.py

```diff
@@ -335,8 +337,8 @@ def load_fields_from_vector(filepath, layer_id=0):
     return fieldnames
 
 
-def check_vector(filepath, fields=None, projected=False, projection_units=None,
-                 **kwargs):
+def check_vector(filepath, geometries, fields=None, projected=False,
```

Comment:
Could you add `geometries` to the Args definition?

Reply:
good catch! Added it to the docstring.